sql/executor.go

```diff
@@ -72,6 +75,8 @@ func (e Executor) Execute(args driver.Request) (driver.Response, int, error) {
 			txn.SetSystemDBTrigger()
 		}
 		planMaker.txn = txn
+		now = planMaker.session.UnixTimestamp.GoTime()
```

Comment:
It's weird that after this point `now` is not the current time. This should use a different local variable `txnTimestamp` (which defaults to `now`).